Scripts/test_set.py

```diff
@@ -0,0 +1,53 @@
+import os
+import random
+import shutil  # importing the required libraries
+
+import cv2
+
+parasite_dir = r"C:\Users\navee\Hakin Codes\cell_images\Parasitized"
```

Comment:
Remove references from your Local System

Reply:
Actually the kaggle dataset is present in my local system (from where I am importing the respective images) and not on my github branch (cause I dont think we need to upload the whole dataset on the repo). Thats why I am not sure as to how to remove it .